### PR TITLE
Upgrade FPE engine and make use of custom alphabets

### PR DESCRIPTION
**Pull request by jarrettranderson**

### CODE REVIEW

**Review by dumashwang — APPROVED** [review]
Looks good.

File: ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fpe/Radix.java [hunk 1]
```diff
@@ -14,24 +14,34 @@ public enum Radix {
...
 
   /**
    * Digits 0-9 and lower case letters a-z
    */
-  DIGITS_LOWER(36, 4, 36, 'a');
+  DIGITS_LOWER("0123456789abcdefghijklmnopqrstuvwxyz", 4, 36, 'a'),
 
-  private final int value;
+  /**
+   * lower case letters A-Z
```
Comment (DenisRicard) [inline ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fpe/Radix.java:30]: Comment should saw upper instead of lower

**Review by DenisRicard — APPROVED** [review]
Looks good. Only one comment that has a cut-and-paste error

File: ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fpe/Radix.java (hunk 1 above)
Comment (jarrettranderson) [reply ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fpe/Radix.java]: good catch - updated

**Pull request merged**